torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1050,6 +1060,25 @@ llvm::Error PyTorchModelLoader::loadMatMul(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], glowNode);
 }
 
+llvmError PyTModelLoaderLoader::LoadReshape(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 2, outputs, 1));
+
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input, getGlowNodeValue(inputs[ReshapeInputs::input]));
+
+  glow::Handle<uint32_t> shapeHandle;
+  ASSIGN_VALUE_OR_RETURN_ERR(shapeHandle, getGlowConstantHandle<uint32_t>inputs[ReshapeInputs::new_shape]);
+  std::vector<size_t> new_shape;
+  ASSIGN_VALUE_OR_RETURN_ERR(new_shape, expandParamIfNeeded<uint32_t, size_t>(shapeHandle, 4));
```

Comment:
this shouldn't be needed, we should take the shape as is and not expect a particular length

Reply:
I wonder how can we do this? Given this shape is always a constant/ListConstruct

Reply:
Not sure what you mean? I'm probably missing something. We import IntLists as glow Constant int tensors, so can't we just use this here without expanding it to a specific size (4)?